### PR TITLE
Wrapper for druid database init scripts

### LINKED ISSUE #13738: Create wrapper scripts for the druid database init scripts
### Description

Per @paul-rogers  comment in #13713 , suggest creating a script to wrap the database initialization scripts to migrate from Derby to Postgres or MySQL

### Motivation

The scripts in `docs/operations/metadata-migration.md` are now mature enough that we don't really need to force the user to check that they copy/pasted this command correctly, and filled in the blanks.

### PR DESCRIPTION
**Pull request by siva-nb**
Fixes #13738

### Description

As described part of the issue, created a wrapper shell script with druid database init scripts

#### Release note

Wrapper for druid database init scripts

This PR has:

- [x] been self-reviewed.
   - [ ] using the [concurrency checklist](https://github.com/apache/druid/blob/master/dev/code-review/concurrency.md) (Remove this item if the PR doesn't have any relation to concurrency.)
- [ ] added documentation for new or modified features or behaviors.
- [x] a release note entry in the PR description.
- [ ] added Javadocs for most classes and all non-trivial methods. Linked related entities via Javadoc links.
- [ ] added or updated version, license, or notice information in [licenses.yaml](https://github.com/apache/druid/blob/master/dev/license.md)
- [ ] added comments explaining the "why" and the intent of the code wherever would not be obvious for an unfamiliar reader.
- [ ] added unit tests or modified existing tests to cover new code paths, ensuring the threshold for [code coverage](https://github.com/apache/druid/blob/master/dev/code-review/code-coverage.md) is met.
- [ ] added integration tests.
- [ ] been tested in a test Druid cluster.

### CODE REVIEW

File: docs/operations/metadata-migration.md [hunk 1]
```diff
@@ -71,15 +71,13 @@ In the example commands below:
 #### MySQL
 
 ```bash
-cd ${DRUID_ROOT}
-java -classpath "lib/*" -Dlog4j.configurationFile=conf/druid/cluster/_common/log4j2.xml -Ddruid.extensions.directory="extensions" -Ddruid.extensions.loadList="[\"mysql-metadata-storage\"]" -Ddruid.metadata.storage.type=mysql -Ddruid.node.type=metadata-init org.apache.druid.cli.Main tools metadata-init --connectURI="<mysql-uri>" --user <user> --password <pass> --base druid
+./_bin/init-metadata.sh -d mysql -u <username> -p <password> -c <mysql-uri>
```
Comment (siva-nb) [inline docs/operations/metadata-migration.md:74]: I am not sure, if the script has to be in this path. Any thoughts on this ?

**Comment by siva-nb** [comment]
@techdocsmith Could you please share your thoughts on this PR ?

**Comment by techdocsmith** [comment]
thanks for the contribution, @siva-nb ! I think @paul-rogers or maybe @findingrish  would be better to comment on this particular pr.

cc: @sergioferragut

File: docs/_bin/init-metadata.sh [hunk 2]
```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
```
Comment (LakshSingla) [inline docs/_bin/init-metadata.sh:1]:
```suggestion
#!/bin/bash -eu
```

File: docs/_bin/init-metadata.sh [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+# Fail fast on any error
+set -e
```
Comment (LakshSingla) [inline docs/_bin/init-metadata.sh:4]:
```suggestion
```
nit: This won't be required once we set `-eu` on the header.

File: docs/_bin/init-metadata.sh [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  echo "Tool for creating Druid's metadata tables"
+  echo
+  echo "Syntax: init-metadata [-d|c|u|p|v]"
+  echo "options:"
+  echo "d     Metadata storage type eg. mysql, postgresql."
+  echo "c     Metadata storage connector connection URI"
+  echo "u     Metadata storage connector username"
+  echo "p     Metadata storage connector password"
+  echo "h     help"
+  echo "example useage init-metadata -d mysql -c localhost -u username -p password"
```
Comment (LakshSingla) [inline docs/_bin/init-metadata.sh:29]:
```suggestion
  echo "example usage init-metadata -d mysql -c localhost -u username -p password"
```

File: docs/_bin/init-metadata.sh [hunk 5]
```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
```
[the remark this answers is not in the document]
Comment (LakshSingla) [reply docs/_bin/init-metadata.sh:2]: Also, can you add a short description of the use of the script below that.

**Review by LakshSingla — COMMENTED** [review]
Hi, @siva-nb, thanks for your first pull request!!

Added a few notes related to the PR. Also, I think a better place for this would be `bin/examples` instead of `docs/_bin`.

**Comment by siva-nb** [comment]
@LakshSingla Thanks for the review, will look into it.

**Pull request closed without merging**